game/scripts/vscripts/internal/eventwrapper.lua

```diff
@@ -0,0 +1,17 @@
+if not _ListenToGameEvent then
+  _ListenToGameEvent = ListenToGameEvent
+end
+
+-- ListenToGameEvent('dota_rune_activated_server', Dynamic_Wrap(GameMode, 'OnRuneActivated'), self)
+function ListenToGameEvent (eventName, fn, obj)
+  return _ListenToGameEvent(eventName, function (...)
+    local data = {...}
+    local status, err = pcall(function()
+      fn(unpack(data))
+    end)
+    if err then
+      print(err)
+      error(err)
```

Comment:
why `print` and `error`?

Reply:
just in case one of the wrappers breaks, since error also throws an exception